### DISCUSSION TITLE
Interfere with firmware to adjust motor speed?

### DISCUSSION THREAD

**Discussion by pdhung197** [pybricks/support#1153] [General]
I have an issue with my Move Hub that the motors A and B have difference speed. As you see in the video, motor A a bit slower than motor B.
I don't know if is there any solution to use pybrick to interfere with firmware to adjust motor speed?

https://github.com/pybricks/support/assets/14067307/938a6769-9413-4992-bf4a-c74ff26ea878

**Comment by dlech** [comment]
If you use Pybricks instead of the official LEGO firmware, then the motors should work better. 

(We can't provide support for the LEGO firmware - you would have to talk to LEGO customer support about it.)
